samples/cpp/facedetect.cpp

```diff
@@ -199,7 +199,7 @@ void detectAndDraw( Mat& img, CascadeClassifier& cascade,
         Size(30, 30) );
     if( tryflip )
     {
-        flip(smallImg, smallImg, 1);
```

Comment:
The following `cascade.detectMultiScale` works with gray-scale images only. img is color and converted at line 188. The patch is definitely invalid and original version works as designed.